tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -237,6 +237,16 @@ endif
 # runtime that can be linked in to other programs.
 MICROLITE_LIB_NAME := libtensorflow-microlite.a
 
+# Where compiled objects are stored.
```

Comment:
If this part is placed here, then `TARGET_ARCH` and `BUILD_TYPE` stored in $(MAKEFILE_DIR)/targets/$(TARGET)_makefile.inc will not be able to affect the `GENDIR` variable, for example, for `TARGET=riscv32_mcu`, the TARGET_ARCH will be default set to x86-64, but before it should be `TARGET_ARCH := riscv32_mcu` defined in https://github.com/tensorflow/tflite-micro/blob/main/tensorflow/lite/micro/tools/make/targets/riscv32_mcu_makefile.inc#L2